nasl/nasl_socket.c

```diff
@@ -1447,9 +1447,9 @@ nasl_get_sock_info (lex_ctxt *lexic)
  * This function is used to retrieve and verify a certificate from an
  * active socket. It requires the NASL socket number.
  *
- * @nasluparam
+ * @naslnparam
  *
- * - A NASL socket.
+ * - @a socket A NASL socket.
  *
  * @naslret 0 in case of successfully verification. A positive integer in
```

Comment:
```suggestion
 * @naslret 0 in case of successfully verification. A negative integer in
```

The `gnutls_x509_trust_list_verify_crt` function description on https://www.gnutls.org/manual/html_node/Verifying-X_002e509-certificate-paths.html currently says the following:

> Returns: On success, GNUTLS_E_SUCCESS (0) is returned, otherwise a negative error value.

Reply:
This has been reverted in f9d443394e34c60d6639596786cf92634f3e0dbe again for the reason given in:

https://github.com/greenbone/openvas-scanner/pull/928#discussion_r745510612

Reply:
The return value of `gnutls_x509_trust_list_verify_crt` is not returned but the value which is put into ` voutput` of the function which is a OR’ed sequence of gnutls_certificate_status_t flags.